pkg/render/helm.go

```diff
@@ -32,5 +32,5 @@ func ToChart(name string, m *manifest.Manifest, options fleet.BundleDeploymentOp
 		return nil, err
 	}
 
-	return m.ToTarGZ()
+	return m.ToTarGZ(options.Decrypt)
```

Comment:
Perhaps we add something above L35 like this...
```go
var decrypt bool
if options.Decrypt == nil {
    decrypt = true
} else {
    decrypt = options.Decrypt
}
```

There may be a more efficient way to do this though.
Regardless, we want to cover the default case: what happens under these three conditions before we execute the `ToTarGZ` method...

1. Unset
2. True
3. False

Here is a playground example: https://play.golang.org/p/TyIFUDuyruG
Here is the plaintext if that link becomes inaccessible:
```go
package main

import (
	"fmt"
)

func example1() {
	var input *bool
	var output bool
	if input == nil {
		output = true
	} else {
		output = *input
	}
	fmt.Printf("%+v\n", output)
}

func example2() {
	var input2 *bool
	var output2 bool

	value := false
	input2 = &value

	if input2 == nil {
		output2 = true
	} else {
		output2 = *input2
	}
	fmt.Printf("%+v\n", output2)
}

func main() {
	example1()
	example2()
}
```

Reply:
Relevant note to consider. This functionality needs to happen, but could be where `GitRepo` CRs are reconciled.
See comments:
- https://github.com/rancher/fleet/pull/298#discussion_r587747643
- https://github.com/rancher/fleet/pull/298#discussion_r587742678